CHANGELOG.rst

```diff
@@ -1,6 +1,30 @@
 Changelog
 =========
 
+1.8.2 (2024-01-15)
+--------------------
+
+Bug Fixes
```

Comment:
```suggestion
Bug Fixes

```

Reply:
the suggestion here and below is to add a new line between the headings and subsequent list items. I guess that's needed, at least the preview was not rendering fine without those blank lines in between.